stubs/boltons/boltons/easterutils.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import NoReturn
+
+def gobs_program() -> NoReturn: ...
```

Comment:
Here `NoReturn` is actually correct

Reply:
Thank you for the clarification on NoReturn and None